src/table/table-row.jsx

```diff
@@ -94,8 +94,9 @@ const TableRow = React.createClass({
 
     let styles = {
       root: {
-        borderBottom: '1px solid ' + this.getTheme().borderColor,
-        color: this.getTheme().textColor,
+        borderBottom: '1px solid ' + theme.borderColor,
+        color: theme.textColor,
+        height: theme.height
```

Comment:
I'm still wondering. What happen if someone put a content that overflow?
Do we have a way to fix this so that the height is linked to the height of the content?